CONTRIBUTING.md

```diff
@@ -13,13 +31,26 @@ please head over to our [feedback page](https://feedback.mongodb.com/forums/9308
 Please create a [GitHub issue](https://github.com/mongodb/mongocli/issues/new?assignees=&labels=&template=bug_report.md) describing the kind of problem you're facing
 with as much detail as possible, including things like operating system or anything else may be relevant to the issue.
 
-## Submitting a Patch
+## Submitting Patches
 
-Before submitting a patch to the repo please consider opening an [issue first](#reporting-issues).
+The MongoDB CLI project welcomes all contributors and contributions regardless of skill or experience level. 
+If you are interested in helping with the project, we will help you with your contribution.
```

Comment:
I really like this statement 💯

Reply:
how about directing them to the guidelines ? 
```suggestion
If you are interested in helping with the project, please follow our [guidelines](#code-contribution-guidelines).
```

Reply:
+1

Reply:
@colm-quinn is the +1 for andrea or for filipe, trying to decide if to keep it as it or not and you are the deciding vote

Reply:
🥁 it's for Filipe :)

Reply:
it's for both @gssbzn, @colm-quinn both likes the comment and the suggestion 😆